python/src/aiconfig/AIConfigSettings.py

```diff
@@ -360,6 +361,60 @@ def delete_prompt(self, prompt_name: str):
         # remove from prompt list
         self.prompts = [prompt for prompt in self.prompts if prompt.name != prompt_name]
 
+    def extract_override_settings(self, inference_settings: InferenceSettings, model_id: str):
```

Comment:
Should this function just be somewhere in utils? It's a bit confusing why it's a core function in this class. Not a huge deal as long as it works properly.

Reply:
moved fn to util file